demo/aws/aws-tomcat/configure-common.tf

```diff
@@ -0,0 +1,15 @@
+# Configure the AWS Provider
+provider "aws" {
+  region     = var.region
+  access_key = var.access_key
+  secret_key = var.secret_key
+}
+
+resource "aws_key_pair" "ubuntu" {
+  key_name   = "ubuntu"
+  public_key = file("key.pub")
+
+  tags = {
+    Name = "failover/tf-ubuntu-key"
```

Comment:
Replace tag name prefix `failover/...` with something more relevant, like `test/..` or `sample/...`. Applies to all tags in this commits.

Reply:
done